src/snac/Config_ddev.php

```diff
@@ -0,0 +1,365 @@
+
+    /**
+     *  @var integer Default for offset in selectListByStatus() and selectEditList()
+     */
+    public static $SQL_OFFSET = 0;
+
+    /**
+     *  @var integer Default for limit selectListByStatus() and selectEditList()
+     */
+    public static $SQL_LIMIT = 42;
+
+    /**
+     * @var string Location of the REST commands file
+     */
+    public static $REST_COMMAND_FILE = "/var/www/html/src/snac/client/rest/commands.json";
+
+    /**
+     * @var string Location of the cpf.rng RELAX NG files, and probably other stuff as well.
+     *
+     * The relative path is probably: vendor/npm-asset/eac-validator/rng
+     */
+    public static $RNG_DIR = "/var/www/html/src/snac/util";
+
+    /**
+     * @var string Location of the EAC-CFP XML Serializer template directory. This is the same directory as
+     * the serializer class.
+     */
+    public static $CPF_TEMPLATE_DIR = "/var/www/html/src/snac/util";
+
+    /**
+     * @var string Location of the template directory
+     */
+    public static $TEMPLATE_LANGUAGE_DIR = "/var/www/html/src/snac/client/webui/languages";
+
+    /**
+     * @var string Location of the template directory
+     */
+    public static $TEMPLATE_DIR = "/var/www/html/src/snac/client/webui/templates";
+
+    /**
+     * @var string Location of the template cache directory
+     */
+    public static $TEMPLATE_CACHE = "/tmp/";
+
+    /**
+     * @var string[][] OAuth connection information.  Each entry should have all the
+     *      required information to connect to that provider.
+     */
+    public static $OAUTH_CONNECTION = array (
+        "google" => array(
+            "client_id" => 'XXXXXXXXX',
+            "client_secret" => 'XXXXXXXX',
+            "redirect_uri" => 'XXXXXXX',
+        )
+    );
+
+    /**
+     * @var int The maximum number of objects allowed in a list of objects.
+     */
+    public static $MAX_LIST_SIZE = 50000;
+
+    /**
+     * @var boolean Whether or not to have the server attempt to use Elastic
+     *      Search indexing.
+     */
+    public static $USE_ELASTIC_SEARCH = true;
+
+    /**
+     * @var string Elastic Search URL
+     */
+    public static $ELASTIC_SEARCH_URI = "http://elasticsearch:9200";
+
+    /**
+     * @var string Main index for the search functionality of snac
+     */
+    public static $ELASTIC_SEARCH_BASE_INDEX = "local";
+
+    /**
+     * @var string All index for the all name (and alternate) search functionality of snac
+     */
+    public static $ELASTIC_SEARCH_ALL_INDEX = "local_all";
+
+    /**
+     * @var string Resource index for the resource search functionality of snac
+     */
+    public static $ELASTIC_SEARCH_RESOURCE_INDEX = "local_resources";
+
+    /**
+     * @var integer Maximum number of pages to show on search result pages
+     */
+    public static $MAX_SEARCH_RESULT_PAGES = 15;
+
+    /**
+     * @var boolean Whether or not to have the server attempt to use Neo4J
+     */
+    public static $USE_NEO4J = true;
```

Comment:
Since we haven't set up the Neo4j db, this line might need to be set to `false` for local development. But if we're not hitting errors so far then it shouldn't be a big deal.